samples/csharp_dotnetcore/51.teams-messaging-extensions-action/Startup.cs

```diff
@@ -26,7 +26,8 @@ public Startup(IConfiguration configuration)
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllers().AddNewtonsoftJson();
-            
+            services.AddRazorPages();
+            services.AddMvc();
```

Comment:
Please review the startup.cs in the TeamsTaskModule sample, which uses Razor Pages

Reply:
Yes done, We have already added razor page service in startup.cs file but we also need MVC service to render Webview as well